src/app/settingswidget/settingswidget.cpp

```diff
@@ -66,13 +66,18 @@ Core::SettingsWidget::SettingsWidget(ExtensionManager *extensionManager,
      */
 
     // HOTKEY
-    QSet<int> hks = hotkeyManager->hotkeys();
-    if (hks.size() < 1)
-        ui.grabKeyButton_hotkey->setText("Press to set hotkey");
-    else
-        ui.grabKeyButton_hotkey->setText(QKeySequence(*hks.begin()).toString()); // OMG
-    connect(ui.grabKeyButton_hotkey, &GrabKeyButton::keyCombinationPressed,
-            this, &SettingsWidget::changeHotkey);
+    if (hotkeyManager) {
+        QSet<int> hks = hotkeyManager->hotkeys();
+        if (hks.size() < 1)
+            ui.grabKeyButton_hotkey->setText("Press to set hotkey");
+        else
+            ui.grabKeyButton_hotkey->setText(QKeySequence(*hks.begin()).toString()); // OMG
+        connect(ui.grabKeyButton_hotkey, &GrabKeyButton::keyCombinationPressed,
+                this, &SettingsWidget::changeHotkey);
+    } else {
+        ui.grabKeyButton_hotkey->setVisible(false);
+        ui.label_hotkey->setVisible(false);
```

Comment:
Design choice:
1. Just hide it
2. Tell the user that it can't be done this way and that they should consult the compositor

I personally would go with option 2

Reply:
You mean like a dialog when you push the button, or a tooltip on a disabled button?

Maybe it can be done separately in another merge request? I'm not sure when I have the time to continue work on this, and in the meantime it would be nice to at least solve the crashes :)